packages/@sanity/portable-text-editor/src/editor/plugins/createWithEditableAPI.ts

```diff
@@ -180,6 +180,24 @@ export function createWithEditableAPI(
           ],
           portableTextEditor,
         )[0] as unknown as Node
+        const [focusBlock] = Array.from(
+          Editor.nodes(editor, {
+            at: editor.selection.focus.path.slice(0, 1),
+            match: (n) => n._type === types.block.name,
+          }),
+        )[0] || [undefined]
+
+        const isEmptyTextBlock =
```

Comment:
You could use the `isEqualToEmptyEditor` helper function here, as it will also check if you did any styling or similar to the block. This is what we use throughout the code for this test.

Reply:
Thanks @skogsmaskin updated